internal/cli/options.go

```diff
@@ -90,6 +95,29 @@ func appConfiguration(cmd *cobra.Command) (models.ApplicationUpdateRequest, erro
 	result.Services = uniqueStrings(services)
 	sort.Strings(result.Services)
 
+	assignments, err := cmd.Flags().GetStringSlice("env")
+	if err != nil {
+		return result, errors.Wrap(err, "failed to read option --env")
+	}
+
+	for _, assignment := range assignments {
+		pieces := strings.Split(assignment, "=")
+		if len(pieces) != 2 {
+			return result, errors.New("Bad --env assignment, expected `name=value` as value")
+		}
+		result.Environment = append(result.Environment, models.EnvVariable{
+			Name:  pieces[0],
+			Value: pieces[1],
+		})
+	}
+
+	// nil --> Default / No change
+	// - AppCreate API will replace it with `v1.DefaultInstances`
+	// - AppUpdate API will treat it as no op, i.e. keep current instances.
```

Comment:
The three preceding comment lines are about the instances, not services, nor EV. Should be moved up to where the instances are handled in this function